index.js

```diff
@@ -1314,6 +1314,8 @@ function Runtime() {
         });
 
         return function (target, data) {
+            if (target.isNull()) 
+                throw new Error("The target protocol is null");
```

Comment:
Yep, that makes sense. How to test this, you mentioned about `npm link` but in node_modules i couldn't find frida-objc. So tbh no idea if you can point out how I would appreciate.
```
[alvaro@alvaro node_modules]$ ls|grep frida
frida
```